front/src/modules/trainschedule/components/Timetable/Timetable.tsx

```diff
@@ -131,45 +123,43 @@ const Timetable = ({
           {t('timetable.addTrainSchedule')}
         </button>
       </div>
-      <TimetableToolbar
-        trainSchedulesWithDetails={trainSchedulesWithDetails}
-        displayedTrainSchedules={displayedTrainSchedules}
-        setDisplayedTrainSchedules={setDisplayedTrainSchedules}
-        selectedTrainIds={selectedTrainIds}
-        setSelectedTrainIds={setSelectedTrainIds}
-        multiSelectOn={multiselectOn}
-        setMultiSelectOn={setMultiselectOn}
-        removeTrains={removeTrains}
-        trainSchedules={trainSchedules}
-      />
-
       <div
         className={cx('scenario-timetable-trains', {
           expanded: conflictsListExpanded,
-          'with-details': trainsWithDetails,
+          'with-details': showTrainDetails,
         })}
       >
+        <TimetableToolbar
+          showTrainDetails={showTrainDetails}
+          toggleShowTrainDetails={toggleShowTrainDetails}
+          trainSchedulesWithDetails={trainSchedulesWithDetails}
+          displayedTrainSchedules={displayedTrainSchedules}
+          setDisplayedTrainSchedules={setDisplayedTrainSchedules}
+          selectedTrainIds={selectedTrainIds}
+          setSelectedTrainIds={setSelectedTrainIds}
+          removeTrains={removeAndUnselectTrains}
+          trainSchedules={trainSchedules}
+        />
         {trainsDurationsIntervals &&
-          displayedTrainSchedules.map((train: TrainScheduleWithDetails, idx: number) => (
+          displayedTrainSchedules.map((train: TrainScheduleWithDetails) => (
             <TimetableTrainCard
-              idx={idx}
-              isSelectable={multiselectOn}
               isInSelection={selectedTrainIds.includes(train.id)}
               handleSelectTrain={handleSelectTrain}
               train={train}
-              intervalPosition={valueToInterval(train.duration, trainsDurationsIntervals)}
               key={`timetable-train-card-${train.id}-${train.trainName}`}
               isSelected={infraState === 'CACHED' && selectedTrainId === train.id}
               isModified={train.id === trainIdToEdit}
-              projectionPathIsUsed={
-                infraState === 'CACHED' && trainIdUsedForProjection === train.id
-              }
               setDisplayTrainScheduleManagement={setDisplayTrainScheduleManagement}
               upsertTrainSchedules={upsertTrainSchedules}
               setTrainIdToEdit={setTrainIdToEdit}
-              removeTrains={removeTrains}
+              removeTrains={removeAndUnselectTrains}
             />
           ))}
+        <div
+          className={cx('bottom-timetables-trains', {
+            'empty-list': trainSchedulesWithDetails.length === 0,
+          })}
+        />
```

Comment:
I think we don't need this div. It just corresponds to the bottom of the `.scenario-timetable-trains` div. It should work with just a padding-bottom and the special `.empty-list` class when no train

Reply:
You're right, I can't remember why I did it but it's not useful. I've just changed it.